### PR TITLE
disable iptables for kube bridges; add forward rules for simple setup

### PR DESCRIPTION
**Pull request by rajatchopra**
@mrunalp review please

### CODE REVIEW

File: ovssubnet/bin/openshift-sdn-simple-setup-node.sh [hunk 1]
```diff
@@ -43,6 +43,9 @@ iptables -D INPUT -i lbr0 -m comment --comment "traffic from docker" -j ACCEPT |
 lineno=$(iptables -nvL INPUT --line-numbers | grep "state RELATED,ESTABLISHED" | awk '{print $1}')
 iptables -I INPUT $lineno -p udp -m multiport --dports 4789 -m comment --comment "001 vxlan incoming" -j ACCEPT
 iptables -I INPUT $((lineno+1)) -i lbr0 -m comment --comment "traffic from docker" -j ACCEPT
+fwd_lineno=$(iptables -nvL FORWARD --line-numbers | grep "reject-with icmp-host-prohibited" tail -n 1 | awk '{print $1}')
```
Comment (sdodson) [inline ovssubnet/bin/openshift-sdn-simple-setup-node.sh]: fwd_lineno=$(iptables -nvL FORWARD --line-numbers | grep "reject-with icmp-host-prohibited" | tail -n 1 | awk '{print $1}')

it's missing the pipe before tail

File: ovssubnet/bin/openshift-sdn-kube-subnet-setup.sh [hunk 2]
```diff
@@ -86,6 +86,9 @@ EOF
     systemctl daemon-reload
     systemctl restart docker.service
 
+    # disable iptables for lbr0
+    sysctl -w net.bridge.bridge-nf-call-iptables=0
```
Comment (mrunalp) [inline ovssubnet/bin/openshift-sdn-kube-subnet-setup.sh]: Hmm, I see that this isn't present on the F21.

File: ovssubnet/bin/openshift-sdn-simple-setup-node.sh (hunk 1 above)
Comment (rajatchopra) [inline ovssubnet/bin/openshift-sdn-simple-setup-node.sh]: copy/paste error; thanks

File: ovssubnet/bin/openshift-sdn-kube-subnet-setup.sh (hunk 2 above)
Comment (mrunalp) [inline ovssubnet/bin/openshift-sdn-kube-subnet-setup.sh]: 1. Do a brctl show, if the setting is not found
2. modprobe br_netfilter

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**